LMBQ-505: Properly register all needed css and js files for prism (#230)
Lombiq/Helpful-Extensions: Lombiq.HelpfulExtensions/Extensions/Trumbowyg/Readme.md

    {
        const string MyContentType = "MyContentType";

        var resourceFilters = new[]
            {
                builder.WhenContentType(MyContentType),
                builder.WhenContentTypeEditor(MyContentType),
                builder.WhenContentTypeCreate(MyContentType),
                builder.WhenContentTypePreview(MyContentType),
            };

        foreach (var resourceFilter in resourceFilters)
        {
            resourceFilter.RegisterStylesheet(Prism, PrismLineHighlight, TrumbowygHighlight);
            resourceFilter.RegisterFootScript(Prism, PrismLineHighlight, TrumbowygHighlight, TrumbowygHighlightExtension);
            foreach (var language in AllLanguage)
            {
                resourceFilter.RegisterFootScript(language);
            }
        }
    }
}
```

This will add the required styles and scripts to the editor and display views of the specified content type. Make sure to replace `MyContentType` with the actual content type name where you want to use this feature. The following languages will be included beyond the base Prism languages:

- C#
- Docker
- Json
- Markdown
- Sql
- Liquid
- Regex
- Powershell

> You can add a new language by registering a resource and including that in a ResourceFilter. It might be dependent on other languages, you can check the dependency by ticking your language [here](https://prismjs.com/download) and looking for other language selected automatically or by checking the language script directly. E.g.:

Reviewer: ```suggestion
You can add a new language by registering its script resource in a Resource Filter. Note that it might have dependencies you need to include, which you can find by inspecting the script, or checking which language gets automatically selected when you select your desired language in the [Prism JS download list](https://prismjs.com/download).
```

```csharp
_manifest
.DefineScript("LiquidLanguage")
.SetUrl(
    Vendors + $"prismjs/components/prism-liquid.min.js",
    Vendors + $"prismjs/components/prism-liquid.js")
.SetDependencies("markup-templating");
```

> You can use the Coy Prism theme by registering the `Lombiq.HelpfulExtensions.Extensions.Trumbowyg.Constants.ResourceNames.PrismCoyTheme` as a stylesheet. The same way as adding new languages you can use any other Prism theme by registering the appropriate resource.

Reviewer: ```suggestion
You can use any other Prism theme by registering their corresponding stylesheet, e.g. `Lombiq.HelpfulExtensions.Extensions.Trumbowyg.Constants.ResourceNames.PrismCoyTheme` for the Coy theme.
```
